README.md

```diff
@@ -328,6 +328,8 @@ For Puppet 4.2.2+ (and 3.8.2) we've lessened the possibility of the issue occurr
 
 If you are affected by this issue, a known workaround is to download the `.tar.gz` from the forge and use `puppet module install` using the file rather than directly from the forge.
 
+- Windows 2003 is not supported.  If installing the module to the default `production` environment on a Puppet master, pluginsync will attempt to install on Windows 2003 nodes within the `production` environment.  This will likely cause pluginsync failures, where the LFN issue is even more problematic (due to a longer user home directory). In this situation, [Puppet environments](https://docs.puppetlabs.com/puppet/latest/reference/environments.html) may be used to segregate nodes and control module installs.  Since modules may be configured independently per environment, this allows for the creation of an environment to isolate Windows 2003, and ensure the DSC module is not installed.
+
```

Comment:
`Windows 2003 is not supported` is the first thing there, so that seems pretty early ;0